src/fontra/client/core/utils.js

```diff
@@ -254,3 +264,16 @@ export async function readFromClipboard(type) {
   }
   return undefined;
 }
+
+function writeToLocalStorage(clipboardObject) {
+  const fallbackTypes = {
```

Comment:
If you write this as a list of lists (a length-two list instead of a key/value pair), the for loop below won't need to `Object.entries` bit.